packages/hash/api/src/model/ontology/data-type.model.ts

```diff
@@ -39,13 +41,51 @@ export default class {
     graphApi: GraphApi,
     params: {
       accountId: string;
-      schema: DataType;
+      // we have to manually specify this type because of 'intended' limitations of `Omit` with extended Record types:
+      //  https://github.com/microsoft/TypeScript/issues/50638
+      //  this is needed for as long as DataType extends Record
+      schema: Pick<DataType, "kind" | "title" | "description" | "type"> &
```

Comment:
Indeed. I think there's _some_ benefit to not having a type alias, as I could see us finally fixing it and then leaving the type alias in place. Whereas here we'd be forced to update all of these to just point to `DataType`. Rather than ending up with something like `type DataType = DataType` and then inconsistently using which.

Reply:
Very fair, I don't think it's worth doing in this case as we do get type information from `DataType`, so at least `Pick` would fail if it's wrong :+1: